components/gitpod-protocol/src/util/tracing.ts

```diff
@@ -13,6 +13,8 @@ import { log, LogContext } from "./logging";
 
 export interface TraceContext {
     span?: opentracing.Span;
+    // TODO(gpl) We are missing this method, but won't add right now because of different focus, and it's unclear how we want to use tracing going forward
+    isDebugIDContainerOnly?: () => boolean;
```

Comment:
is that used somewhere or required? The comment confuses me a bit.

Reply:
I can improve the message. It's required by SpanContext, which we are using it for, but somehow not hit on most paths with the current version of the opentracing library version we are using.
I hit that problem in one of our tests, but was able to workaround it for now. Just a hint for the next one running into this.